EasyCommand.AspNetCore.Tests/EasyCommandServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using EasyCommand.AspNetCore.Tests.Commands;
+using EasyCommand.AspNetCore.Tests.Handler;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using System;
+using Xunit;
+
+namespace EasyCommand.AspNetCore.Tests
+{
+    public class EasyCommandServiceCollectionExtensionsTests
+    {
+        [Fact]
+        public void SuccessfullyRegistersCommand_NoOptions()
+        {
+            var services = CreateServiceProvider(
+                    t => t.AddEasyCommand());
+
+            var command = services.GetService<ExampleCommandWithRequestAndResult>();
```

Comment:
Should this not use this.Command<T> for completeness? Assuming this.Command may, in time, mutate the command before returning it.

Reply:
in this instance we are just checking that a command is registered in the DI container, if we are to use the this.Command then we will be checking the ControllerBaseExtensions as well and we would also need to build the service as well to provide a controller.